src/routes/quickstarts/react/test-a-component-with-react-testing-library.mdx

```diff
@@ -30,7 +31,10 @@ import { createServer } from "miragejs"
 let server
 
 beforeEach(() => {
-  server = createServer()
+  server = createServer({
+    environment: "test",
+    urlPrefix: "http://localhost:3000",
```

Comment:
could you make this a comment, and have the example say something like `api.acme.com`? And move the "`urlPrefix` is only required if your API is at a different port or host" comment here to the comment as well?

Reply:
done